### PR TITLE
 Postfix Evaluation

### PR DESCRIPTION
**Pull request by muthuannamalai12**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Added C code for Postfix Evaluation
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#2045
(Write your answer here.)

### CODE REVIEW

File: C/ds/EvaluationOfPostfixExpression.c [hunk 1]
```diff
@@ -0,0 +1,77 @@
+/*
+A postfix expression is a collection of operators and operands in which the operator is placed after the operands. That means, in a postfix expression the operator follows the operands.
+A postfix expression can be evaluated using the Stack data structure. To evaluate a postfix expression using Stack data structure we can use the following steps...
+1.Read all the symbols one by one from left to right in the given Postfix Expression
+2.If the reading symbol is operand, then push it on to the Stack.
+3.If the reading symbol is operator (+ , - , * , / etc.,), then perform TWO pop operations and store the two popped oparands in two different variables (operand1 and operand2). Then perform reading symbol operation using operand1 and operand2 and push result back on to the Stack.
+4.Finally! perform a pop operation and display the popped value as final result.
```
Comment (HarshCasper) [inline C/ds/EvaluationOfPostfixExpression.c]: Modularize this. It is tough to read and skim through on Mobile/Laptop Devices

File: C/ds/EvaluationOfPostfixExpression.c [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+#include <stdio.h>
+#include <unistd.h>
+struct stack
+{
+    int top;
+    float a[50];
+}
+s;
+void main()
```
Comment (HarshCasper) [inline C/ds/EvaluationOfPostfixExpression.c]: Use `int main()` instead of `void`

File: C/ds/EvaluationOfPostfixExpression.c [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                break;
+            case '*':
+                d2 = s.a[s.top--];
+                d1 = s.a[s.top--];
+                s.a[++s.top] = d1 * d2;
+                break;
+            case '/':
+                d2 = s.a[s.top--];
+                d1 = s.a[s.top--];
+                s.a[++s.top] = d1 / d2;
```
Comment (HarshCasper) [inline C/ds/EvaluationOfPostfixExpression.c]: Few changes: 
- Use meaningful variable names 
- Shift the Stack Code to a function

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by muthuannamalai12** [comment]
I have updated kindly review again @HarshCasper @atarax665

File: C/ds/EvaluationOfPostfixExpression.c [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+#include <unistd.h>
+struct stack
+{
+    int top;
+    float a[50];
+}
+s;
+int main()
+{
+    char ch[50];
```
Comment (atarax665) [inline C/ds/EvaluationOfPostfixExpression.c]: Use a meaningful name for variables. i.e -  postfix[30]  etc. Do the same with variables x, y, z.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by muthuannamalai12** [comment]
I have updated kindly review again @atarax665

**Comment by muthuannamalai12** [comment]
I have updated kindly review again @atarax665

**Review by atarax665 — CHANGES_REQUESTED** [review]
Shift the stack code in a function. The main function should only have the driver code.

**Comment by muthuannamalai12** [comment]
I have updated it. Kindly Review again @atarax665

**Review by HarshCasper — APPROVED** [review]

**Comment by muthuannamalai12** [comment]
@HarshCasper @atarax665. Can you please add DWOC and level label for this issue here on my PR.

**Comment by muthuannamalai12** [comment]
@HarshCasper @atarax665 @anushkrishnav . Can you please add DWOC and level label for this issue here on my PR.

**Pull request merged**